mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -1507,7 +1508,29 @@ struct BlockwiseGemmRewritePattern : public OpRewritePattern<miopen::BlockwiseGe
     //     ThreadwiseGemmTransANormalBNormalC<decltype(a_thread_mtx),
     //                                        decltype(b_thread_mtx),
     //                                        decltype(c_thread_mtx)>{};
-    lb.create<miopen::ThreadwiseGemmOp>(op.getLoc(), threadAAllocOp, threadBAllocOp, op.getOperand(2));
+
+    // TODO: Update hard-coded gemmA/B/C shape to actual shapes when
+    // threadwisecopy done
+    ArrayRef<int64_t> threadAShape = {8, 16};
+    auto threadA2DMemRefType =
+        MemRefType::get(threadAShape, b.getF32Type(), {}, registerMemorySpace);
+    auto threadA2DViewOp = lb.create<ViewOp>(op.getLoc(), threadA2DMemRefType,
```

Comment:
@jerryyin do we really need these ViewOp?

Reply:
I can get rid of them. The threadwisegemm lowering op is assuming 2d dimension and the viewOp is just a way to feed fake 2d memref. Once you have the dimension right, I can do a rebase and get rid of the ViewOp.